ext/uri/tests/rfc3986/modification/fragment_success_encoded.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Test Uri\Rfc3986\Uri component modification - fragment - URL encoded characters
+--EXTENSIONS--
+uri
+--FILE--
+<?php
+
+$uri1 = Uri\Rfc3986\Uri::parse("https://example.com");
+$uri2 = $uri1->withFragment("foo%3Dbar"); // foo=bar
```

Comment:
Should this also have a normalizable percent-encoded character similarly to `ext/uri/tests/rfc3986/modification/host_success_encoded.phpt`?